packages/react/src/app-layout/docs/overview.mdx

```diff
@@ -65,15 +65,25 @@ The app layout sidebar should be hidden while users are completing multi-page fo
 
 ### Sub-level items
 
-To help users comprehend the side navigation, choose a strategy that is appropriate for the number of items in the list.
+When choosing if sub-level items should be always visible or only visible when active you should consider:
```

Comment:
```suggestion
When choosing if sub-level items should be always visible or only visible when active you should consider:

```

Reply:
Our styling linter is getting tripped because it wants an empty line before the start of a list. Same with the one below.